packages/components/datagrid/src/js/datagrid.controller.js

```diff
@@ -420,6 +421,13 @@ export default class DatagridController {
     return this.expandedRows.includes(index);
   }
 
+  onRowExpand() {
```

Comment:
what is this function used for ?

Reply:
(it only returns true or null)

Reply:
@frenautvh its an optional method, we need this function to do certain operation like making api call on expand row or any other functionality only on row expand.